Wappscript.js

```diff
@@ -0,0 +1,71 @@
+const windSpeedMap = new Map();
+const temperatureMap = new Map();
+
+
+fetch("http://localhost:8000/windspeed.json")
+            .then(response => response.json())
+            .then(data => {
+                fetch("http://localhost:8000/air-temperature.json")
+                    .then(response => response.json())
+                    .then(temperatureData => {
+                        
+                        const debugElement = document.querySelector("#debug");
+
+            
+
+                        for (let i = 0; i < data.items[0].readings.length; i++) {
+                            const windSpeedReading = data.items[0].readings[i];
+                            const temperatureReading = temperatureData.items[0].readings[i];
+
+                            const windSpeedStation = data.metadata.stations.find(station => station.id === windSpeedReading.station_id);
+                            const temperatureStation = temperatureData.metadata.stations.find(station => station.id === temperatureReading.station_id);
+
+                            const windSpeedStationName = windSpeedStation ? windSpeedStation.name : "Unknown Station";
+                            const temperatureStationName = temperatureStation ? temperatureStation.name : "Unknown Station";
+
+                            const windSpeedValue = windSpeedReading.value;
+                            const temperatureValue = temperatureReading.value;
+
+                            windSpeedMap.set(windSpeedStationName, windSpeedValue);
+                            temperatureMap.set(temperatureStationName, temperatureValue);
+
+
+                                
+                        
+                        }
+                        
+                            Sortbytemperature(temperatureMap);
```

Comment:
I believe there are extra spaces in front of these functions. Let's remove them to correct the indentations.